llvm/test/CodeGen/AMDGPU/GlobalISel/regbankselect-mui-rb-legalize.mir

```diff
@@ -0,0 +1,858 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn -mcpu=gfx1010 -run-pass=none %s -verify-machineinstrs -o - | FileCheck %s
```

Comment:
Shouldn't this run rb-legalize? It only returns true, so it should be pretty harmless.

Reply:
true, will update mir test run lines

Reply:
Sorry, correction, not true. Machine verifier will complain that Generic virtual register must have a bank in a RegBankSelected function.
Now that I think of it, RBLegalize should probably also require that MachineFunction has Property::RegBankSelected

Reply:
I'm not sure what the point of this test is then. If it's testing the verifier, it should go in test/MachineVerifier (and then doesn't need -verify-machineinstrs)